presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -584,52 +615,70 @@ private List<List<?>> insertNulls(List<Type> types, List<List<?>> values)
     public void assertRoundTrip(Type type, List<?> readValues)
             throws Exception
     {
-        assertRoundTrip(type, type, readValues, readValues, true, ImmutableList.of());
-    }
-
-    public void assertRoundTripWithSettings(Type type, List<?> readValues, List<OrcReaderSettings> settings)
-            throws Exception
-    {
-        assertRoundTrip(type, type, readValues, readValues, true, settings);
+        assertRoundTripWithSettings(type, type, readValues, readValues, true, ImmutableList.of());
     }
 
-    public void assertRoundTrip(Type type, List<?> readValues, List<Map<Integer, Map<Subfield, TupleDomainFilter>>> filters)
+    public void assertRoundTrip(
+            Type type,
+            List<?> readValues,
+            List<Map<Integer, Map<Subfield, TupleDomainFilter>>> filters,
+            List<Map<Integer, Map<Subfield, TupleDomainFilter>>> valuesFilters)
             throws Exception
     {
-        List<OrcReaderSettings> settings = filters.stream()
-                .map(entry -> OrcReaderSettings.builder().setColumnFilters(entry).build())
-                .collect(toImmutableList());
+        ImmutableList.Builder<OrcReaderSettings> settingsBuilder = ImmutableList.builder();
+        for (int i = 0; i < filters.size(); i++) {
+            settingsBuilder.add(OrcReaderSettings.builder().setColumnFilters(filters.get(i)).setExpectedValuesFilters(valuesFilters.get(i)).build());
```

Comment:
reformat this, extract a new variable for OrcReaderSettings and put each builder operation on a new line

Reply:
Sure, reformatted to
```
        ImmutableList.Builder<OrcReaderSettings> settingsBuilder = ImmutableList.builder();
        OrcReaderSettings.Builder orcSettingBuilder = OrcReaderSettings.builder();
        for (int i = 0; i < filters.size(); i++) {
            settingsBuilder.add(orcSettingBuilder
                    .setColumnFilters(filters.get(i))
                    .setExpectedValuesFilters(valuesFilters.get(i))
                    .build());
        }
```